HubFramework.xcodeproj/project.pbxproj

```diff
@@ -2222,31 +2232,27 @@
 		8ABD6CAE1DF6EC36005BCB33 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				HUB_NON_ERROR_WARNINGS_0720 = "";
-				HUB_NON_ERROR_WARNINGS_0730 = "$(HUB_NON_ERROR_WARNING_0720) -Wno-error=partial-availability";
 				INFOPLIST_FILE = tests/Info.plist;
 				PRODUCT_BUNDLE_IDENTIFIER = com.spotify.HubFrameworkTests;
 				PRODUCT_NAME = HubFrameworkTests;
 				WARNING_CFLAGS = (
 					"$(inherited)",
-					"$(HUB_NON_ERROR_WARNINGS_$(XCODE_VERSION_MINOR))",
 					"-Wno-documentation-unknown-command",
+					"-Wno-partial-availability",
```

Comment:
This is for test targets when you import `XCTest` right? I'm pretty sure you can do this:

```objc
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wpartial-availability"
#import <XCTest/XCTest.h>
#pragma clang diagnostic push
```

I think you can just slap that in its own header like `HUBXCTest.h` then just import that header at the top of your test classes.